mass_editing/__openerp__.py

```diff
@@ -20,32 +20,25 @@
 #
 ##############################################################################
```

Comment:
Please change to new OCA header https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py#L3